ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -10179,17 +10111,6 @@ periodics:
     job-release: "4.12"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-hypershift-install-p2-f21
-  reporter_config:
```

Comment:
I think these settings should not be removed.

Reply:
not sure why it is getting removed ..